core/contracts/oracle/implementation/Governor.sol

```diff
@@ -113,11 +113,11 @@ contract Governor is MultiRole, Testable {
 
     /**
      * @notice Executes a proposed governance action that has been approved by voters.
-     * @dev This can be called by any address.
+     * @dev This can be called by any address. Caller is expected to send enough ETH to execute payable transactions.
      * @param id unique id for the executed proposal.
      * @param transactionIndex unique transaction index for the executed proposal.
      */
-    function executeProposal(uint id, uint transactionIndex) external {
+    function executeProposal(uint id, uint transactionIndex) external payable {
```

Comment:
Should we check to ensure that `msg.value == transaction.value` to make sure the user doesn't accidentally send in more than the transaction will use?

Reply:
Yeah that's reasonable

Reply:
Although, it becomes tricky to figure out exactly how much ETH to send, any ideas?

Reply:
I guess user could determine how much ETH to send by examining the public `Transaction` struct using the `id` and `transactionId`. 

Hurts UX but worth it so Governor doesn't hold on to excess ETH

Reply:
Yeah, exactly, I was thinking the same thing. But since it's hardcoded when the proposal is created, it's not that bad just to look up how much ETH is expected. For anything that requires a dynamically determined amount of ETH, the governor setup wouldn't really work in the first place I guess.